libsql/internal/ws/driver.go

```diff
@@ -86,21 +86,29 @@ func convertToNamed(args []driver.Value) []driver.NamedValue {
 }
 
 func (s stmt) Exec(args []driver.Value) (driver.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), defaultWSTimeout)
-	defer cancel()
-	return s.c.ExecContext(ctx, s.query, convertToNamed(args))
```

Comment:
Maybe just call s.ExecContext(ctx, args)?

Reply:
(explained earlier for s.QueryContext)